src/components/card/card.js

```diff
@@ -8,7 +8,8 @@
 angular.module('material.components.card', [
     'material.core'
   ])
-  .directive('mdCard', mdCardDirective);
+  .directive('mdCard', mdCardDirective)
+  .directive('mdPlainImage', mdPlainImageDirective);
```

Comment:
Correct me if I am wrong, but that would not be backward's compatible right?

Is it better to just make a breaking change?